docs/topics/configuration.md

```diff
@@ -70,3 +75,19 @@ configuration file, for example by writing `MAX_PODS=200 krustlet-wascc` or
 If you specify node labels in multiple places, the collections are _not_
 combined: the place with the highest precedence takes effect and all others
 are ignored.
+
+## Notes to kubelet implementers
+
+Some flags require you to support them in your provider or main code - they are
+not implemented automatically by the kubelet core. These flags are as follows:
+
+* `--bootstrap-file` - should be passed to `kubelet::bootstrap` if you use the
+  bootstrapping feature
+* `--data-dir` - this should be used to construct the `FileStore` if you use one
+* `--allow-local-modules` - if specified you should compose a `FileSystemStore`
+  onto your normal store
+
+See the `krustlet-wasi.rs` file for examples of how to honour these flags.
+
+If you can't honour a flag value in your particular scenario, then you should
+still check for it and return an error, rather than silently ignoring it.
```

Comment:
👍 I like the call to action here for developers on how to properly handle this behaviour.